wordpress.org/public_html/wp-content/themes/pub/wporg-breathe/functions.php

```diff
@@ -51,6 +51,11 @@ function no_robots( $noindex ) {
 		$noindex = true;
 	}
 
+	// This is used by https://github.com/WordPress/phpunit-test-reporter/blob/master/src/class-display.php on the test reporter page
+	if ( isset( $_GET['rpage'] ) && $_GET['rpage'] > 1 ) {
```

Comment:
```suggestion
	if ( isset( $_GET['rpage'] ) ) {
```

I think we can just check to see if it's set, since the ?rpage=1 url shouldn't really exist, but is "incorrectly" linked to by the page navigation.